Nodejs/Tests/Core/Debugger/Commands/SetBreakpointCommandTests.cs

```diff
@@ -24,6 +26,37 @@ namespace NodejsTests.Debugger.Commands {
     [TestClass]
     public class SetBreakpointCommandTests {
         [TestMethod, Priority(0), TestCategory("Debugging")]
+        public void CreateLocalScriptRegExpShouldCreateCaseInSensitiveRegularExpression() {
+            const string drive = "c:";
+            var pathParts = new[] { "nOdE", "IS", "awsome.js" };
```

Comment:
This test was actually a meta test designed to test the ability of the programmer to spell...

Should be fixed now.